src/serializer/MapFishPrintV3GeoJsonSerializer.js

```diff
@@ -395,13 +400,27 @@ export class MapFishPrintV3GeoJsonSerializer extends BaseSerializer {
       return {};
     }
 
+    // If not set, getLineDash will return null.
+    // according to http://mapfish.github.io/mapfish-print-doc/styles.html#mapfishJsonParser
+    // strokeDashstyle - (line, point, polygon) A string describing how to draw the line or an array of floats
+    // describing the line lengths and space lengths:
+
+    // dot - translates to dash array: [0.1, 2 * strokeWidth]
+    // dash - translates to dash array: [2 * strokeWidth, 2 * strokeWidth]
+    // dashdot - translates to dash array: [3 * strokeWidth, 2 * strokeWidth, 0.1, 2 * strokeWidth]
+    // longdash - translates to dash array: [4 * strokeWidth, 2 * strokeWidth]
+    // longdashdot - translates to dash array: [5 * strokeWidth, 2 * strokeWidth, 0.1, 2 * strokeWidth]
+    // {string containing spaces to delimit array elements} - Example: [1 2 3 1 2]
+
+    // olStrokeStyle.getLineDash() will return a number array, wich is not valid for mapfish print.
+    // Here we translate the number array in lineDash to a valid value recognized by mapfish print like "5 5" instead
+    // of [5, 5]
     return {
       color: olStrokeStyle.getColor(),
       lineCap: olStrokeStyle.getLineCap(),
       lineJoin: olStrokeStyle.getLineJoin(),
-      // If not set, getLineDash will return null.
-      lineDash: olStrokeStyle.getLineDash() || undefined,
-      lineDashOffeset: olStrokeStyle.getLineDashOffset(),
+      lineDash: olStrokeStyle.getLineDash().toString().replace(/\,/g, ' ') || undefined,
```

Comment:
if `olStrokeStyle.getLineDash()` returns null here, `toString()` method will lead to an error

Reply:
Ops, yes, you are absolutely correct!